.eslintrc.js

```diff
@@ -10,26 +10,37 @@ module.exports = {
     requireConfigFile: false,
   },
   rules: {
-    // allow reassigning param
-    'no-param-reassign': [2, { props: false }],
-    'linebreak-style': ['error', 'unix'],
+    'chai-friendly/no-unused-expressions': 2,
     'import/extensions': ['error', { js: 'always' }],
+    'linebreak-style': ['error', 'unix'],
+    'no-await-in-loop': 0,
+    'no-param-reassign': [2, { props: false }],
+    'no-restricted-syntax': [
+      'error',
+      {
+        selector: 'ForInStatement',
+        message: 'for..in loops iterate over the entire prototype chain, which is virtually never what you want. Use Object.{keys,values,entries}, and iterate over the resulting array.',
+      },
+      {
+        selector: 'LabeledStatement',
+        message: 'Labels are a form of GOTO; using them makes code confusing and hard to maintain and understand.',
+      },
+      {
+        selector: 'WithStatement',
+        message: '`with` is disallowed in strict mode because it makes code impossible to predict and optimize.',
+      },
+    ],
+    'no-return-assign': ['error', 'except-parens'],
+    'no-unused-expressions': 0,
     'object-curly-newline': ['error', {
       ObjectExpression: { multiline: true, minProperties: 6 },
       ObjectPattern: { multiline: true, minProperties: 6 },
       ImportDeclaration: { multiline: true, minProperties: 6 },
       ExportDeclaration: { multiline: true, minProperties: 6 },
     }],
-    'no-return-assign': ['error', 'except-parens'],
-    'no-unused-expressions': 0,
-    'chai-friendly/no-unused-expressions': 2,
-
   },
   overrides: [
-    {
-      files: ['test/**/*.js'],
-      rules: { 'no-console': 'off' },
-    },
+    { files: ['test/**/*.js'] },
```

Comment:
Since there's no longer an override for this path, is it worth keeping it?

Reply:
eh probably not but I'm going to leave it just in case at some point we want to add other test folders